config/locales/en.yml

```
@@ -33,4 +33,13 @@ en:
   active_record:
     errors:
       note:
-        review_too_long: "has too many words, reviews have a limit of %{limit} words for %{utility_name}"
\ No newline at end of file
+        review_too_long: "a review cant have more than %{limit} words for %{utility_name}"
```

```suggestion
        review_too_long: "a review cant have more than %{limit} words for %{utility_name}."
```
Mismo para el resto!